lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -71,6 +73,7 @@
                         ['topics/:topic_id/search(/)', _.bind(this.searchTeams, this)],
                         ['topics/:topic_id/create-team(/)', _.bind(this.newTeam, this)],
                         ['topics/:topic_id/:team_id/edit-team(/)', _.bind(this.editTeam, this)],
+                        ['topics/:topic_id/:team_id/edit-team/manage-members(/)', _.bind(this.editTeamMembers, this)],
```

Comment:
I agree. If it's an easy fix to remove them from the router, that's no problem, but otherwise I think handling server errors in the linked ticket will be good enough.